### PR TITLE
refactor: Fix compilation warnings in Java part

### LINKED ISSUE #697: Fix All the Compilation Warnings in Java Part
## Description ##
The server contains a lot of Java compilation warnings. They should be fixed when possible.

## Acceptance Criteria ##
- All the data bus warnings are fixed
- All the `CobolErrorStrategy` warnings are fixed
- All the raw `Class` calls are parametrized
- All the unused imports and fields are removed
image (10)

## Definition of Done ##
- The story is implemented according to the description.
- Unit testing done (Preferably by TDD approach, coverage should be at least 70%).
- Code review done by at least one other team member.
- Manual testing is done.
- Documentation is added and reviewed (if needed).
- The pull request is approved.
- The story is merged into the development branch.
- Tests are performed on the development branch.

## Technical details ##
The Guice warning is a platform limitation.

### PR DESCRIPTION
**Pull request by grianbrcom**
Java code check style was added.
Raw Class usages was fixed.
Style improvements.

Closes #697

Signed-off-by: Anton Grigorev <anton.grigorev@broadcom.com>

### CODE REVIEW

**Comment by grianbrcom** [comment]
DataBus part require a separate refactoring.

File: server/src/style/checkstyle-suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```
Comment (temanbrcom) [inline server/src/style/checkstyle-suppressions.xml:1]: Shouldn't we have a copyright header here?

**Comment by temanbrcom** [comment]
When is the style check applied? Is it a part of the build?

**Comment by grianbrcom** [comment]
This style check applied at the "validate" step before "compile" by maven. So if your code has an unused import or missed copyright header it will not be compiled.

File: server/src/style/checkstyle-suppressions.xml (hunk 1 above)
Comment (grianbrcom) [reply server/src/style/checkstyle-suppressions.xml:1]: Fixed

**Review by temanbrcom — APPROVED** [review]
🐝

**Review by asatklichov — APPROVED** [review]
❄️

**Pull request merged**